.github/workflows/meta.yml

```diff
@@ -37,18 +37,17 @@ jobs:
   circular:
     uses: plone/meta/.github/workflows/circular.yml@main
 
-##
-# To modify the list of default jobs being created add in .meta.toml:
-# [github]
-# jobs = [
-#    "qa",
-#    "test",
-#    "coverage",
-#    "dependencies",
-#    "release_ready",
+# Modify the list of default jobs being created add in .meta.toml:
```

Comment:
This commit should not have been done here in `.github/workflows/meta.yml`, but in `.meta.toml` in the root of the repo.  And then run `plone/meta` over it again to update the configuration.
Now the GitHub Actions workflow file has syntax errors.

I fix this here: https://github.com/collective/plone.edu/pull/19